### ISSUE TITLE
Issue With iOS 15.6.1

### ISSUE THREAD

**Issue by rlee1990** [SimformSolutionsPvtLtd/audio_waveforms#86]
I get the below error on ios: 
```scheduledTimer(withTimeInterval:repeats:block:)' is only available in iOS 10.0 or newer```
By switching:
```
func startListening(){
        timer = Timer.scheduledTimer(withTimeInterval: 0.2, repeats: true, block: {_ in
            let ms = (self.player?.currentTime ?? 0) * 1000
            self.plugin.onCurrentDuration(duration: Int(ms),playerKey: self.playerKey)
        })
    }```
to:

```
func startListening(){
        if #available(iOS 10.0, *) {
            timer = Timer.scheduledTimer(withTimeInterval: 0.2, repeats: true, block: {_ in
                let ms = (self.player?.currentTime ?? 0) * 1000
                self.plugin.onCurrentDuration(duration: Int(ms),playerKey: self.playerKey)
            })
        } else {
            // Fallback on earlier versions
        }
    }
```
it then builds

**Comment by rlee1990** [comment]
I will submit a pull request.

**Comment by ujas-m-simformsolutions** [comment]
@rlee1990  Can you please also add `flutter doctor` logs

**Comment by DhavalRKansara** [comment]
Closing the issue as the changes have already been merged with PR #87. Thanks for the contribution @rlee1990
